Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Build pxz in separate image to avoid big image size
-FROM ubuntu:19.10 AS build
-RUN apt-get update && apt-get install -y \
+FROM ubuntu:20.04 AS build
+RUN apt-get update && env DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Why not `Dockerfile` `ENV` syntax?

Reply:
Yes, I suppose that is better, if a new apt command is added later. Will change it.

Reply:
On second thought, maybe not. The Dockerfile reference [explicitly recommends not setting `DEBIAN_FRONTEND=noninteractive` using `ENV`](https://docs.docker.com/engine/reference/builder/#env). We are doing this in a lot of other places though, but I guess we can at least follow the advice here.

Reply:
Sure. Still, we don't really have interactive users of our image though

Reply:
Interesting that this actually found a place in the documentation :p